scripts/upgrade

```diff
@@ -64,6 +64,20 @@ ynh_setup_source "$final_path"
 # NGINX CONFIGURATION
 #=================================================
 
+# Delete current nginx configuration to be able to check if .well-known is already served.
+ynh_remove_nginx_config
+ynh_app_setting_delete $app "checksum__etc_nginx_conf.d_$domain.d_$app.conf" || true
```

Comment:
Here, could you please check if the config has been modified by the user before deleting it.
https://github.com/YunoHost/yunohost/blob/unstable/data/helpers.d/filesystem#L285

Reply:
I took the code from here https://github.com/YunoHost-Apps/nextcloud_ynh/blob/master/scripts/upgrade#L109-L116 and do understand what and why it is needed

Reply:
I don't know myself, but it would be good to use the helper `ynh_backup_if_checksum_is_different file`, in case the admin changed its config file.